coops-ui/tests/unit/StandaloneDirectorsFiling.spec.ts

```diff
@@ -400,6 +329,81 @@ describe('Standalone Directors Filing - Part 3 - Submitting', () => {
     sinon.restore()
   })
 
+  it('enables File & Pay button when Validated is true', () => {
```

Comment:
I don't think these tests should have been moved - they're UI checks (button enable/disable) rather than functions of submitting.

Reply:
Sounds good, great point. I will adjust!